projection/transaction/transaction.go

```diff
@@ -72,25 +72,41 @@ func (projection *Transaction) HandleEvents(height int64, events []event_entity.
 			blockTime = blockCreatedEvent.Block.Time
 			blockHash = blockCreatedEvent.Block.Hash
 		} else if transactionCreatedEvent, ok := event.(*event_usecase.TransactionCreated); ok {
-			txs = append(txs, transaction_view.TransactionRow{
-				BlockHeight:   height,
-				BlockTime:     utctime.UTCTime{}, // placeholder
-				Hash:          transactionCreatedEvent.TxHash,
-				Index:         transactionCreatedEvent.Index,
-				Success:       true,
-				Code:          transactionCreatedEvent.Code,
-				Log:           transactionCreatedEvent.Log,
-				Fee:           transactionCreatedEvent.Fee,
-				FeePayer:      transactionCreatedEvent.FeePayer,
-				FeeGranter:    transactionCreatedEvent.FeeGranter,
-				GasWanted:     transactionCreatedEvent.GasWanted,
-				GasUsed:       transactionCreatedEvent.GasUsed,
-				Memo:          transactionCreatedEvent.Memo,
-				TimeoutHeight: transactionCreatedEvent.TimeoutHeight,
-				Messages:      make([]transaction_view.TransactionRowMessage, 0),
-			})
+			if transactionCreatedEvent != nil {
```

Comment:
Why do we need a nil check here?

Reply:
Should be false positive on lint, already removed